src/components/campaigns/CampaignFilter.tsx

```diff
@@ -0,0 +1,255 @@
+import { IconButton, ImageList, Typography } from '@mui/material'
+import makeStyles from '@mui/styles/makeStyles'
+import createStyles from '@mui/styles/createStyles'
+import { useCampaignList, useCampaignTypesList } from 'common/hooks/campaigns'
+import AddIcon from '@mui/icons-material/Add'
+import SpaIcon from '@mui/icons-material/Spa'
+import BeachAccessIcon from '@mui/icons-material/BeachAccess'
+import FavoriteIcon from '@mui/icons-material/Favorite'
+import CampaignsList from './CampaignsList'
+import { CampaignResponse } from 'gql/campaigns'
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    filterButtons: {
+      display: 'block',
+      height: '100px',
+      borderRadius: 0,
+      '&:active': {
+        color: '#4AC3FF',
+        borderBottom: '5px solid #4AC3FF',
+      },
+      '&:hover': {
+        backgroundColor: 'white',
+        borderBottom: '5px solid #4AC3FF',
+        color: '#4AC3FF',
+      },
+      '&:focus': {
+        color: '#4AC3FF',
+        borderBottom: '5px solid #4AC3FF',
+      },
+      '&:selected': {
+        color: '#4AC3FF',
+        borderBottom: '5px solid #4AC3FF',
+      },
+    },
+  }),
+)
+
+export default function CampaignsPage() {
+  const classes = useStyles()
+  const { data: campaignTypes } = useCampaignTypesList()
+  const { data: campaigns } = useCampaignList()
+  const [campaignType, setCampaignType] = useState('All')
+  const [campaignToShow, setCampaignToShow] = useState<CampaignResponse[]>([])
+  let natureCounter = 0
+  let healthCareCounter = 0
+  let disastersCounter = 0
+  let othersCounter = 0
+
+  campaignTypes?.map(() => {
+    natureCounter = 0
+    healthCareCounter = 0
+    disastersCounter = 0
+    othersCounter = 0
+    campaigns?.map((campaign) => {
+      campaignTypes?.map((type) => {
+        if (campaign.campaignTypeId == type.id) {
+          if (
+            type.name == 'Leukemia' ||
+            type.name == 'Transplantation' ||
+            type.name == 'Cancer' ||
+            type.name == 'Nursing homes' ||
+            type.name == 'Treatment and rehabilitation' ||
+            type.name == 'Rehabilitation' ||
+            type.name == 'Genetic diseases'
+          ) {
+            healthCareCounter++
+          }
+          if (type.name == 'Nature' || type.name == 'Animals') {
+            natureCounter++
+          }
+          if (type.name == 'Disasters') {
+            disastersCounter++
+          }
+          if (
+            type.name != 'Disasters' &&
+            type.name != 'Nature' &&
+            type.name != 'Animals' &&
+            type.name != 'Treatment and rehabilitation' &&
+            type.name != 'Leukemia' &&
+            type.name != 'Nursing homes' &&
+            type.name != 'Transplantation' &&
+            type.name != 'Cancer' &&
+            type.name != 'Genetic diseases' &&
+            type.name != 'Rehabilitation'
+          ) {
+            othersCounter++
+          }
```

Comment:
We should never depend on the actual name of the campaign as it would be different when translated in Bulgarian.

We need to find different way of differentiating those categories, even if we need to add an additional db field